kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/WaitUntilReadyExample.java

```diff
@@ -48,14 +50,33 @@ public static void main(String args[]) throws IOException, InterruptedException
         .endSpec()
         .build();
 
+      Pod secondPod = new PodBuilder()
```

Comment:
It's not a second pod that is missing from this test, but more like a resource like a Secret or something, which is not waitable.